jupyterlab/labbox_ephys_widgets_jp/package.json

```diff
@@ -11,6 +11,8 @@
   "files": [
     "lib/**/*.js",
     "dist/*.js",
+    "lib/**/*.css",
```

Comment:
Had some trouble with the new css files during building of the jupyterlab extension